### PR TITLE
es6modules

### PR DESCRIPTION
**Pull request by easton-morris**

### CODE REVIEW

File: es6-modules/src/components/todo-app.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+
+
+import TodoList from './todo-list';
+import TodoForm from './todo-form';
+import createElement from '../lib/create-element';
+import noop from '../lib/noop';
```
Comment (scbowler) [inline es6-modules/src/components/todo-app.js]: Here you want to `import` these from the `lib/index.js` file:
```JAVASCRIPT
import { createElement, noop } from '../lib';
```

File: es6-modules/src/components/todo-form.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import createElement from '../lib/create-element';
```
Comment (scbowler) [inline es6-modules/src/components/todo-form.js]: Here you want to `import` this from the `lib/index.js` file:
```JAVASCRIPT
import { createElement } from '../lib';
```

File: es6-modules/src/components/todo-list.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import createElement from '../lib/create-element';
```
Comment (scbowler) [inline es6-modules/src/components/todo-list.js]: Here you want to `import` this from the `lib/index.js` file:
```JAVASCRIPT
import { createElement } from '../lib';
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but a few updates needed. Let me know if you have any questions.

File: es6-modules/src/components/todo-app.js (hunk 1 above)
Comment (easton-morris) [reply es6-modules/src/components/todo-app.js]: updated

File: es6-modules/src/components/todo-form.js (hunk 2 above)
Comment (easton-morris) [reply es6-modules/src/components/todo-form.js]: updated

File: es6-modules/src/components/todo-list.js (hunk 3 above)
Comment (easton-morris) [reply es6-modules/src/components/todo-list.js]: updated

File: es6-modules/src/components/todo-app.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
+
+
+import TodoList from './todo-list';
+import TodoForm from './todo-form';
+import  { createElement } from '../lib/index';
+import { noop } from '../lib/index';
```
Comment (scbowler) [inline es6-modules/src/components/todo-app.js:6]: You can do this with a single import statement, and you don't need to specify the `index` file.
```JAVASCRIPT
import { createElement, noop } from '../lib';
```

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍 

1 FYI comment, no changes required.

**Pull request merged**